azure/deploy-to-azure/azure-integration/CreateUIDefinition.json

```diff
@@ -251,6 +251,17 @@
                 "type": "Microsoft.Common.CheckBox",
                 "label": "Collect Custom Metrics",
                 "toolTip": "This option will enable custom metric collection from all Application Insights instances within the scope of this configuration. These custom metrics will appear in datadog under the namespace <b>application_insights.custom.&lt;METRIC_NAME&gt;</b>.<br> Note that <a target='_blank' href='https://learn.microsoft.com/en-us/azure/azure-monitor/app/standard-metrics/'>Standard</a> App Insights metrics are collected automatically and are available under the <b>azure.insights_components.*</b> namespace."
+              },
+              {
+                "name": "excluded_resource_providers",
+                "type": "Microsoft.Common.TextBox",
+                "label": "Excluded Resource Providers",
+                "defaultValue": "",
+                "toolTip": "Comma-separated list of Microsoft Resource Provider names to be excluded from metrics collection.",
```

Comment:
same comment as before about the tooltip

Reply:
done